utils/evaluate_utils.py

```diff
@@ -99,6 +99,149 @@ def class_PR_curves(
     return fig, PR_data
 
 
+def class_PR_curves_SCM(
+    single_cell_data: pd.DataFrame,
+    single_class_model: LogisticRegression,
+    fig: Figure,
+    axs: np.ndarray,
+    phenotypic_class_index: int,
+    data_split_colors: dict,
+    model_type: str,
+    feature_type: str,
```

Comment:
Based on what's found in the docstring below, consider using [`typing.Literal`](https://docs.python.org/3.8/library/typing.html#typing.Literal) here to specify certain static strings (instead of any string).

Reply:
I've never seen this before, thanks for teaching me something new! Added in https://github.com/WayScience/phenotypic_profiling_model/commit/987e1ec5a50e2751ae962ec53d06265164fcb301